helm-values/velero.yaml

```diff
@@ -2,32 +2,35 @@ fullnameOverride: ${fullname_override}
 
 %{ if role_enabled }
 serviceAccount:
-  annotations:
-    eks.amazonaws.com/role-arn: ${role_arn}
-    eks.amazonaws.com/sts-regional-endpoints: "${sts_regional_endpoints}"
+  server:
+    create: true
+    annotations:
+      eks.amazonaws.com/role-arn: ${role_arn}
+      eks.amazonaws.com/sts-regional-endpoints: "${sts_regional_endpoints}"
 %{ endif }
 
+credentials:
+  useSecret: false
+
 configuration:
-  provider: aws
   backupStorageLocation:
-    bucket: ${bucket_id}
-    name: default
-    prefix: format("backups/%s", ${eks_cluster_id})
+  - name: 
     provider: aws
-    region: ${region}
+    bucket: ${bucket_id}
+    prefix: ${eks_cluster_id}
+    accessMode: ReadWrite
+    config:
+      region: ${region}
+      kmsKeyId: ${kms_key_id}
+
   volumeSnapshotLocation:
-    name: default
+  - name: 
```

Comment:
from docs
`# name is the name of the volume snapshot location where snapshots are being taken. Required.`